modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -47,12 +47,20 @@ def veteran
       @veteran ||= Appellant.new(
         type: :veteran,
         auth_headers: auth_headers,
-        form_data: data_attributes&.dig('veteran')
+        form_data: data_attributes['veteran']
+      )
+    end
+
+    def claimant
```

Comment:
Added to support signing_appellant. We could hard code it as `veteran`, but I liked getting it set up knowing we're adding claimant soon.